src/core/token/TokenManager.ttslua

```diff
@@ -161,52 +161,39 @@ do
   ---@param card Object Card to spawn tokens on
   ---@param tokenType String type of token to spawn, valid values are "damage" and "horror".  Other
   --     types should use spawnMultipleTokens()
-  ---@param tokenValue Value to set the damage/horror to
-  ---@param shiftDown An offset for the z-value of this group of tokens
+  ---@param tokenValue Number Value to set the damage/horror to
   TokenManager.spawnCounterToken = function(card, tokenType, tokenValue, shiftDown)
-    if tokenCount < 1 or tokenCount > 50 then
-      return
-    end
-    local offsets = PLAYER_CARD_TOKEN_OFFSETS[1]
-    if shiftDown ~= nil then
-      -- Copy the offsets to make sure we don't change the static values
-      local baseOffsets = offsets
-      offsets = { }
-      for i, baseOffset in ipairs(baseOffsets) do
-        offsets[i] = baseOffset
-        offsets[i][3] = offsets[i][3] + shiftDown
-      end
-    end
-    local pos = card.positionToWorld(offsets[1])
-    pos.y = card.getPosition().y + 0.15
-    TokenManager.spawnToken(pos, tokenType, card.getRotation(), function(spawned)
+    if tokenValue < 1 or tokenValue > 50 then return end
+
+    local pos = card.positionToWorld(card.positionToLocal(card.getPosition()) + Vector(0, 0.2, shiftDown))
```

Comment:
Can remove the interior card.positionToLocal(card.getPosition()) here - that's just going to give the origin of the card.  positionToWorld(Vector(0, 0.2, shiftDown)) is sufficient